database/seeders/PermissionsSeeder.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tipoff\Waivers\Database\Seeders;
+
+use Carbon\Carbon;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\Schema;
+
+class PermissionsSeeder extends Seeder
+{
+    public function run()
+    {
+        if (Schema::hasTable('permissions')) {
```

Comment:
Is this necessary?

Reply:
If use of permissions is to be optional, then yes this is required -- in theory, this project could be used by some other project that doesn't have permissions.  If we make use of permissions an explicit dependency, then these seeders will be redone using `Permissions` contract methods.